pkg/test/resourcefixture/testdata/stateabsentinspec/bigquerydataset/_http.log

```diff
@@ -291,8 +389,61 @@ X-Xss-Protection: 0
 
 ---
 
-DELETE https://bigquery.googleapis.com/bigquery/v2/projects/${projectId}/datasets/${datasetID}?alt=json&deleteContents=false
-Content-Type: application/json
+GET https://bigquery.googleapis.com/bigquery/v2/projects/${projectId}/datasets/${datasetID}?alt=json&prettyPrint=false
```

Comment:
Going through all the diffs, I found that there is an additional GET right before DELETE. Is it intentional? If so, why?